ibis/mapd/operations.py

```diff
@@ -573,6 +573,20 @@ def _table_column(translator, expr):
 count = _reduction('count')
 
 
+def _arbitrary(translator, expr):
+    arg, how, where = expr.op().args
+
+    if where is not None:
+        arg = where.ifelse(arg, ibis.NA)
+
+    if how not in (None, 'last'):
```

Comment:
Perform this check before the `if where is not None` check.

Reply:
sounds good.